tensorflow-sources.file

```diff
@@ -118,10 +120,13 @@ cp -p $srcdir/compiler/tf2xla/lib*.so* $libdir/
 cp -p $srcdir/compiler/xla/lib*.so* $libdir/
 
 for l in tensorflow_cc tensorflow_framework tensorflow ; do
-  if [ ! -e $libdir/lib${l}.so ] ; then
-    [ -e  $libdir/lib${l}.so.%{realversion} ] || exit 1
-    ln -s lib${l}.so.%{realversion} $libdir/lib${l}.so
-  fi
+  # link from majorversion to realversion
```

Comment:
can you please add the following here. Just to make sure that full version library exists and it is a file
```
[ -f  $libdir/lib${l}.so.%{realversion} ] || exit 1
```